### DISCUSSION TITLE
Meeting 1 Agenda 1/11/2022

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/omi-scripting-group#8] [General]
# OMI WASM Group Meeting 1

Our first meeting will be on 1/11/2022 at 5:00 PM (UTC) / 9:00 AM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-wasm-group channel to manage a speaker queue, post links, and for any sidebar discussions.

If you missed the original discussion around forming this group and people's introductions, it's availiable [here](https://github.com/omigroup/omi-wasm-group/discussions/4).

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-wasm-group role in the #roles channel of the OMI Discord.

## Agenda
- Attendance
- Introductions (name, short bit on your background)
- Go around the room and each person can have 2 minutes to talk about what they would like to see this group accomplish
- Open floor for responses and general discussion with the objective of starting to come to consensus on the goals of this group. Chat queue will be managed in this discord chat.

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by mrmetaverse** [comment]
Hey everyone, I'd like to add one item: 

- how often will we meet like this? (weekly, bi-weekly, etc.) 
- do you want to plan for any recurring demos to look forward to and show progress? (i.e. a demo session open to OMI every 4-6 weeks, etc.) 
- do you want to plan to have a recurring retrospective, so that the team can share what went well, what didn't, and what to do differently next time? (usually every 4-6 weeks as well) 

Upon identification of your meeting cadence, we can add to the OMI event calendar. <3

**Reply by mrmetaverse** [reply]
keep in mind, if you don't do retro, you're not actually iterating. Many people aren't comfortable bringing up issues or pain points ad-hoc. A retro is a super quick way to share thoughts, and make iterative intentional improvements. This is a big part of our core values, and one that I hope this group will adopt. I will volunteer to run every retro for this group until someone else wants to take over.

**Comment by LightLodges** [comment]
Agenda item to discuss: community engagement & events cadence

It would be GREAT to HOST & PARTICIPATE in more community events in 2022. We have some budget and capacity to bring together more demo nights, livestreams, meetups, hackathons, remote hack together events or participate in a wide variety of other people's events in 2022. What is most important for THIS COMMUNITY to provide and support for all of us to thrive?

Current event thoughts:  
Quarterly demo nights (last one was an evening stream in September, we're due for another soon)
Quarterly meetups/physical gatherings with events (last one was during AWE, ad hoc during virtual events/pandemic)
Weekly conversations online - interviews, Twitter spaces, metaverse events/tours, participating in other people's communities
Hack together sessions: best cadence? GOALS? 
Hackathon -- do we partner with others like WebXR or make one happen on our own later in the year to support OMI R&D work and move projects that may be stalling or need incentives to be aligned with people/actions? Summer 2022?

Want to have dinner together indoors or out after it is healthy to do so? Let's make sure we're connected as human beings who live in physical spaces and can help each other out as needs arise. Regional reachout R&D free zones to meet up for future collaboration sessions in 4-7 months?
